### ISSUE TITLE
scheduler-cli issue (hibernate flag not being read)

### ISSUE THREAD

**Issue by ibroxtech** [awslabs/aws-instance-scheduler#135]
Downloaded latest version of instance scheduler cli from :

https://s3.amazonaws.com/solutions-reference/aws-instance-scheduler/latest/scheduler-cli.zip

Deployed two periods successfully :

 {
         "Name": "test1", 
         "Endtime": "02:00", 
         "Type": "period", 
         "Weekdays": [
            "mon"
         ], 
         "Begintime": "00:00"
      }, 
      {
         "Name": "test2", 
         "Endtime": "04:00", 
         "Type": "period", 
         "Weekdays": [
            "tue"
         ], 
         "Begintime": "00:00"
      }, 

Try to create schedule I get :

scheduler-cli create-schedule --stack XXXXXXXXXXX --name testtest --periods test1,test2 --timezone Australia/Sydney

Error: hibernate is not a valid parameter, valid parameters are ['timezone', 'periods', 'name', 'description', 'overwrite', 'use_metrics', 'stop_new_instances', 'use_maintenance_window', 'retain_running', 'enforced', 'override_status', 'configured_in_stack']

Reading the documentation the default value for --hibernate is false (if you dont include in the command).

If I modify the source code and comment the following :

grep -r hibernate *
scheduler_cli/scheduler_cli.py:PARAM_HIBERNATE = "--hibernate"
scheduler_cli/scheduler_cli.py:                                     dest='hibernate',

grep -r PARAM_HIBERNATE *
scheduler_cli/scheduler_cli.py:PARAM_HIBERNATE = "--hibernate"
scheduler_cli/scheduler_cli.py:        schedule_parser.add_argument(PARAM_HIBERNATE,

Then run the same command, it works :

scheduler-cli create-schedule --stack XXXXXXXXXX --name testtest --periods test1,test2 --timezone Australia/Sydney
{
   "Schedule": {
      "RetainRunning": false, 
      "Enforced": false, 
      "Name": "testtest", 
      "UseMetrics": false, 
      "StopNewInstances": true, 
      "Periods": [
         "test1", 
         "test2"
      ], 
      "UseMaintenanceWindow": false, 
      "Timezone": "Australia/Sydney", 
      "Type": "schedule"
   }
}

Is this expected behaviour or a bug ?

**Comment by tapughose** [comment]
Hello @ibroxtech, thanks for your query. I don't see you used `hibernate` param in your command that you have posted. please let us know the exact command that you were trying to execute by using `hibernate` param.

**Comment by ibroxtech** [comment]
Hi @tapughose it looks like I've found the issue.  The version of the scheduler that was deployed in this particular tenancy was an old version, so I was using the new version of scheduler-cli against an older version of the instance scheduler that was deployed. I've since corrected this and upgraded the CloudFormation stack within the tenancy and it looks like scheduler-cli is now working without any of my modifications :).

I'll close this now.

Thanks for your help.
